### ISSUE TITLE
No screenreader feedback when using disaggregation controls on chart/table

### ISSUE THREAD

**Issue by brockfanning** [open-sdg/open-sdg#1656]
**Please complete the following information** 
Site repository URL: N/A
Open SDG version: 2.0.0-dev

**Describe the bug**

Previously we had functionality where screenreaders would announce any changes to the chart/table after the disaggregation controls had been used, but this seems to have stopped working.

**To Reproduce**
1. Go to any indicator with disaggregation
2. Make sure a screenreader is running
3. Change something with the disaggregation controls
4. Note there is no announcement

**Expected behavior**

Each change should have an informative announcement.

**Additional context**

This was previously implemented in #875
